src/oc-id/config/environments/production.rb

```diff
@@ -27,7 +27,7 @@
   # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = false
+  config.assets.compile = true
```

Comment:
I did run through with node uninstalled - but I'll verify that if it's never ther for the precompile phase, it still behaves just in case something got generated and retained.